examples/jsm/controls/OrbitControls.js

```diff
@@ -581,17 +596,88 @@ class OrbitControls extends EventDispatcher {
 
 		}
 
-		function handleMouseWheel( event ) {
+		function getTotalOffsetLeftAndTop(dom){
 
-			if ( event.deltaY < 0 ) {
+			let left = 0, top = 0;
+			getOffset(dom);
+
+			function getOffset(dom){
+				left += dom.offsetLeft;
+				top += dom.offsetTop;
+				if(dom.offsetParent){
+					getOffset(dom.offsetParent)
+				}
+			}
+
+			return {
+				top: top,
+				left: left
+			}
+		}
+
+		function followZoom(event){
+
+			let x = (event.offsetX / scope.domElement.clientWidth) * 2 -1,
+			y = -(event.offsetY / scope.domElement.clientHeight) * 2 + 1,
+			v = new Vector3(x, y, 0);
+
+			v.unproject(scope.object);
+			v.sub(scope.object.position).setLength(scope.zoomSpeed * 10) // it needs more speed here;
+			
+			if(scope.object.type === "PerspectiveCamera"){
+
+				if(event.deltaY < 0){
+					scope.object.position.add(v);
+					scope.target.add(v);
+				} else {
+					scope.object.position.sub(v);
+					scope.target.sub(v);
+				}
+			}else if(scope.object.type === "OrthographicCamera"){
+
+				const dom = scope.domElement;
+				const domOffset = getTotalOffsetLeftAndTop(dom);
+				const centerX = dom.clientWidth * 0.5;
+				const centerY = dom.clientHeight * 0.5;
```

Comment:
How about using `dom.getBoundingClientRect()` to get the dom-rect and offset at the same time?